src/main/java/decodes/cwms/resevapcalc/EvapMetData.java

```diff
@@ -8,74 +8,62 @@
 
 import decodes.db.Constants;
 import decodes.tsdb.CTimeSeries;
-import decodes.tsdb.IntervalCodes;
 //import hec.heclib.util.HecTime;
 //import hec.io.TimeSeriesContainer;
 //import rma.util.RMAConst;
 
 import java.util.Date;
-import java.util.Objects;
 
 /**
  *  Class hold meteorological time series data for Resevap program.
  */
 public class EvapMetData
 {
     // input met data timeseries
-    CTimeSeries _windspeedTsc;
-    CTimeSeries _airTempTsc;
-    CTimeSeries _relHumidityTsc;
-    CTimeSeries _dewPointTsc;
-    CTimeSeries _airPressureTsc;
-    CTimeSeries _fractionLowClouds;
-    CTimeSeries _altitudeLowClouds;
-    CTimeSeries _fractionMedClouds;
-    CTimeSeries _altitudeMedClouds;
-    CTimeSeries _fractionHighClouds;
-    CTimeSeries _altitudeHighClouds;
+    CTimeSeries windspeedTsc;
+    CTimeSeries airTempTsc;
+    CTimeSeries relHumidityTsc;
+    CTimeSeries dewPointTsc;
+    CTimeSeries airPressureTsc;
+    CTimeSeries fractionLowClouds;
+    CTimeSeries altitudeLowClouds;
+    CTimeSeries fractionMedClouds;
+    CTimeSeries altitudeMedClouds;
+    CTimeSeries fractionHighClouds;
+    CTimeSeries altitudeHighClouds;
     
     // these variables hold last valid values
     //was originally RMA undefinedDouble = -FloatMax
-    double _wsTemp_old = Constants.undefinedDouble;
-    double _windSpeed_old = Constants.undefinedDouble;
-    double _airTemp_old = Constants.undefinedDouble;
-    double _relHumidity_old = Constants.undefinedDouble;
-    double _airPressure_old = Constants.undefinedDouble;
+    double wsTemp_old = Constants.undefinedDouble;
+    double windSpeed_old = Constants.undefinedDouble;
+    double airTemp_old = Constants.undefinedDouble;
+    double relHumidity_old = Constants.undefinedDouble;
+    double airPressure_old = Constants.undefinedDouble;
     
     // test variables hold the met values for the current time
-    double _wsTemp_current = Constants.undefinedDouble;
-    double _windSpeed_current = Constants.undefinedDouble;
-    double _airTemp_current = Constants.undefinedDouble;
-    double _relHumidity_current = Constants.undefinedDouble;
-    double _airPressure_current = Constants.undefinedDouble;
+    double wsTemp_current = Constants.undefinedDouble;
```

Comment:
camel case.

should probably expand ws, not sure what it means in this context (looking at nothing else.)